solr/core/src/java/org/apache/solr/filestore/DistribFileStore.java

```diff
@@ -223,15 +219,12 @@ private boolean fetchFileFromNodeAndPersist(String fromNode) {
         filedata.reset();
         persistToFile(filedata, metadata);
         return true;
-
-      } catch (SolrServerException | IOException e) {
+      } catch (SolrServerException e) {
         throw new SolrException(SolrException.ErrorCode.SERVER_ERROR, "Error fetching data", e);
+      } catch (IOException ioe) {
+        throw new SolrException(SERVER_ERROR, "Error persisting file", ioe);
```

Comment:
notice the needless lopsidedness in both throw statements.  Use the import.

+1 to your changes today BTW.